CareKitUI/CareKitUI/Shared/Style/OCKColorStyler.swift

```diff
@@ -113,11 +126,23 @@ public extension OCKColorStyler {
     var tertiaryCustomFill: UIColor { #colorLiteral(red: 0.462745098, green: 0.462745098, blue: 0.5019607843, alpha: 0.24) }
     var quaternaryCustomFill: UIColor { #colorLiteral(red: 0.462745098, green: 0.462745098, blue: 0.5019607843, alpha: 0.18) }
 
+    #elseif os(macOS)
+
+    var secondaryCustomGroupedBackground: Color { .white }
```

Comment:
I chose `.white` for this and `tertiaryCustomFill`, let me know if you think a better color should be used for macOS